server/views/includes/index-header-nav.pug

```diff
@@ -0,0 +1,16 @@
+nav.section.navigation
+    ul.navigation-bar
+        li
+            a(href="/") Home
+        li
+            a(href="/#link-to-novels") Novels
+        li
+            a(href="/#link-to-stories") Short Stories
+        li
+            a(href="/#link-to-articles") Stories inspired by
+        li
+            a(href="/#link-to-articles") Articles
+        li
+            a(href="/#link-to-poetry") Haiku Poetry
+        li
+            a(href="/#link-to-contacts") Contacts
```

Comment:
Missing new line :)

Reply:
Thank you, will know that I need an extra empty line at the end of the file. And I've updated the VS code with the extension you suggested.